src/compositions/useFavicon.ts

```diff
@@ -1,9 +1,33 @@
 import { onUnmounted, watchEffect, Ref, ref } from 'vue'
 import { StateType } from '@/models/StateType'
 
+function getPreferredColorScheme(): 'dark' | 'light' | 'no-preference' {
+  if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
+    return 'dark'
+  } if (window.matchMedia('(prefers-color-scheme: light)').matches) {
+    return 'light'
+  }
+  return 'no-preference'
+}
```

Comment:
Should this be based on the prefect-design mode? That way if a user switches their color settings this reflects that change.

Reply:
Nope, this is a system setting - the background of the tab/window is what's important, not the application